collections/_article/review-godotcon24.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "GodotCon 24: Berlin Edition"
+excerpt: "Our biggest conference yet!"
+categories: ["events"]
+author: Nat
+image: /storage/blog/covers/godotcon-2024-post.webp
+date: 2024-11-11 18:30:00
+---
+
+<br>
+📋 [Feedback form for visitors](https://forms.gle/3tALsQid32qqvu2Z8)
+🎤 [Talk & workshop recordings](https://www.youtube.com/playlist?list=PLeG_dAglpVo6TS0q858NajyeglRuvb7hs)
+🫙 [Jam games](https://godotcon.itch.io/)
+<hr>
+
+Variety was the name of the game this year. GodotCon keeps going around the globe and adds more activities to the program: the event team made up of volunteers and Godot Foundation staff put a lot work into making this more than a mere meetup occasion for the ever-growing community forming around the Godot Engine. 
+
+Judging by the smiley pictures taken all around the [Silent Green](https://www.silent-green.net/) venue, and the excited chatter about what the next installments might bring to the table, we seem to have hit the sweet spot for our 574 attendees. For those of you that couldn’t be there yourself - considering both ticket batches sold out within an hour - join us on a retrospective walk through GodotCon, as well as highlighting all the wonderful speakers, exhibitors, and sponsors that made this event possible.
+
+# Sponsor support
+GodotCon 2024 wouldn’t have been possible without the generosity of [our sponsors](https://conference.godotengine.org/2024/#sponsors), who enabled us to bring this event to life.
+
+## Critical Reflex
+Having a publisher with experience in Godot on-board was as exciting for us as it seemed for the audience of “Indie Publishing 101” by their business developer Daniel. Additionally, you might have spotted the goodie bags they provided us with.
+
+🔗 [criticalreflex.com](https://criticalreflex.com)
+
+<img alt="Venue picture featuring Critical Reflex goodie bag" src="/storage/blog/godotcon24/goodie-bag.webp"/>
+
+## Endless Foundation
+“Block Coding: Making Godot Accessible to Learners Everywhere” was another well-visited talk on the agenda, as was the booth of the biggest exhibitor team onsite. Their tech demonstration drew in fellow educators as well as curious game developers looking to contribute.
+
+🔗 [endlessos.org](https://www.endlessos.org/)
+
+<img alt="Screenshot of their block coding editor" src="/storage/blog/godotcon24/block-coding.webp"/>
+
+## W4 Games
+Port your Godot games to console with ease! The company founded by Godot Engine veterans explained “W4's role in the Godot Ecosystem” on stage and let you playtest their products & talk business at their busy booth. 
+
+🔗 [w4games.com](https://w4games.com)
+
+<img alt="" src="/storage/blog/godotcon24/w4-games.webp"/>
+
+## Poki
+The provider of free online games also has shown interest in Godot before and is hoping to expand their selection with your projects. We talked with them about improving Godot for the Web together, let’s see what the future might bring!
```

Comment:
```suggestion
The provider of free online games also has shown interest in Godot before and is hoping to expand their selection with your projects. We talked with them about improving Godot for the Web together; let’s see what the future might bring!
```
Needs a bigger break than just a comma. Alternatively, if you want the comma, need to add a word after it like "so".